tests/pkcli/ci_test.py

```diff
@@ -5,17 +5,31 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from pykern.pkdebug import pkdp
+import re
 import pytest
 
+
 def test_run():
     from pykern.pkcli import ci
     from pykern import pkio
     from pykern import pkunit
 
-    for d in pkunit.case_dirs():
+    for d in pkunit.case_dirs("case"):
         try:
             ci.run()
             res = "run ok\n"
         except Exception as e:
             res = f"run failed={e}\n"
         pkio.write_text('res', res)
+
+
+def test_check_prints():
+    from pykern.pkcli import ci, test
+    from pykern import pkio
+    from pykern import pkunit
+    from pykern import pksetup
+
+    r = re.compile( f"/{test.SUITE_D}/.*{pkunit.DATA_DIR_SUFFIX}/|/{pksetup.PACKAGE_DATA}/|pkdebug")
```

Comment:
This bypasses a big part of the code. How would you test exclude but still allow this test to function? (Hint: pkunit.module_under_test, which maybe could be better encapsulated.)

Reply:
I'm not sure I follow you. I looked at the module_under_test in pykern code and it was not clear to me what that is doing.